From: Jakub Jirutka <jakub@jirutka.cz>
Date: Thu, 7 Apr 2016 21:06:00 +0200
Subject: [PATCH] Download Ivy to the current directory instead of home

We don't want to clutter up home directory on the build server, do we?
---
diff --git a/build.xml b/build.xml
index 841582e..2571f46 100644
--- a/build.xml
+++ b/build.xml
@@ -56,8 +56,8 @@
   </target>
 
   <target name="-ivy-install" unless="ivy.available">
-    <mkdir dir="${user.home}/.ant/lib" />
-    <get src="http://repo1.maven.org/maven2/org/apache/ivy/ivy/${ivy.jar.version}/${ivy.jar.name}" dest="${user.home}/.ant/lib/${ivy.jar.name}"/>
+    <mkdir dir=".ant" />
+    <get src="http://repo1.maven.org/maven2/org/apache/ivy/ivy/${ivy.jar.version}/${ivy.jar.name}" dest=".ant/${ivy.jar.name}"/>
   </target>
 
   <target name="install-cpptasks" unless="cpptasks.available">
